### PR TITLE
✨ NeoMessage component

### LINKED ISSUE #5683: Rmrk minting some dialog
Seems this needs right bg color? 

https://canary.kodadot.xyz/ksm/create/

image

plus found some errors in console

image

### PR DESCRIPTION
**Pull request by roiLeo**
## PR Type

- [x] Refactoring

## Context

- [x] related #5852 
- [x] closed #5683

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

## Screenshot 📸

Screenshot 2023-05-22 at 17-13-24 KodaDot Low fees and low carbon minting KodaDot - NFT Market Explorer

### <samp>🤖 Generated by Copilot at f92b398</samp>

Refactored the token creation feature to use a custom `NeoMessage` component from the `@kodadot1/brick` library instead of the Buefy `b-message` component. Deleted unused or redundant components and added documentation and testing for the new component.

### <samp>🤖 Generated by Copilot at f92b398</samp>

> _Sing, O Muse, of the skillful refactorer_
> _Who from the Buefy `b-message` component_
> _Crafted a new and splendid `NeoMessage`_
> _And placed it in the `@kodadot1/brick` library._

### CODE REVIEW

**Comment by vikiival** [comment]
Copy-pasted logic from b-message ?

**Comment by roiLeo** [comment]
> Copy-pasted logic from b-message ? 

Oui.

**Comment by roiLeo** [comment]
> > Oui.
> 
> `Error while parsing: Viki parser is unable to decode emotional context behind the message`

Low quality effort, it's working but at what cost?

**Comment by roiLeo** [comment]
> seems to be working fine in the deployed version, but i wasn't able to reproduce the error using the link provided on #5683

I don't know maybe it was fixed in another PR, I've just render component in random pages to see if it was working.

> @roiLeo can you provide more screenshots of what has been changed or it was just this specific message?

In this PR you can test the `NeoMessage` by adding `?congratsNft=congrats` to any Nft Item

example:
https://deploy-preview-6068--koda-canary.netlify.app/bsx/gallery/4104196707-39

Note: I know it's not looking good, this is a Refactoring PR no design changes has been made.

**Comment by prury** [comment]
> > seems to be working fine in the deployed version, but i wasn't able to reproduce the error using the link provided on #5683
> 
> I don't know maybe it was fixed in another PR, I've just render component in random pages to see if it was working.
> 
> > @roiLeo can you provide more screenshots of what has been changed or it was just this specific message?
> 
> In this PR you can test the `NeoMessage` by adding `?congratsNft=congrats` to any Nft Item
> 
> example: https://deploy-preview-6068--koda-canary.netlify.app/bsx/gallery/4104196707-39?congratsNft=congrats
> 
> Note: I know it's not looking good, this is a Refactoring PR no design changes has been made.



> 

Thank you! 
What event should trigger this message in the future? a NFT mint? 

Apart from the design side then, i found one bug on mobile(resized browser), if you click on the QR code button and leave it open for long enough for the timer to hide the modal, it locks the page scroll.

**Comment by roiLeo** [comment]
> Thank you! What event should trigger this message in the future? a NFT mint?

Yes! keep in mind this is going to change in (can't find realted issue)

> Apart from the design side then, i found one bug on mobile(resized browser), if you click on the QR code button and leave it open for long enough for the timer to hide the modal, it locks the page scroll.

if you're able to reproduce it on this preview and beta, then we can create a new issue.
If it's only comming from my changes, I can take a look.

File: components/shared/gallery/JustMintedTokenMessage.vue
Comment (vikiival) [inline components/shared/gallery/JustMintedTokenMessage.vue:1]: Why delete?

Comment (roiLeo) [reply components/shared/gallery/JustMintedTokenMessage.vue:1]: Not used

**Review by vikiival — APPROVED** [review]

**Review by preschian — APPROVED** [review]

**Pull request merged**